extensions/target-specific/chromium/blink/fragment-tree/fragment-tree.js

```diff
@@ -43,13 +59,8 @@ Loader.OnLoad(function() {
                             "You may still specify a blink::NGPhysicalFragment explicitly.";
                         return Promise.reject(errorMessage);
                     }
-                    // We have to get the child of the root layout object (the LayoutView) because LayoutView never has a fragment.
-                    // All documents have a LayoutView, so we don't have to nullcheck that.
-                    let firstChildrenPromise = Promise.map(documents,
-                        (document) => document.F("node_layout_data_").f("layout_object_").vcast().f("children_.first_child_").vcast())
-                        .then((array) => array.filter((layout_object) => !layout_object.isNull()));
-                    return Promise.map(firstChildrenPromise,
-                        (layout_object) => layout_object.f("cached_layout_result_.ptr_.physical_fragment_.ptr_"));
+                    return Promise.map(documents,

```

Comment:
This is the exact same code that's present in the LayoutObjectTree extension. Can you write a shared helper for it in BlinkHelpers, similar to GetDocuments().

Reply:
Done